### ISSUE TITLE
Broken runKey

### ISSUE THREAD

**Issue by OnceThenEver** [gdh1995/vimium-c#366]
from 1.90.2 of macOS in Chrome:

```
map f runKey expect="sala:<v-sala>" keys="<v-nonsala>"

map <v-nonsala> LinkHints.activateModeToOpenInNewForegroundTab autoUnhover newtab="force-mode"
map <v-sala> LinkHints.activate antoUnhover

env sala host="chrome-extension://此处为Sala的ID/"
```
My code takes all web as the same indiscriminately, even if the host is replaced with "https://www.github.com/"

**Comment by michielappelman** [comment]
Same issue with 1.90.2 for Firefox:

```
map F runKey expect={"hn":"<v-fhn>"} keys="<v-sfd>"
map <v-sfd> LinkHints.activateModeToOpenInNewTab
map <v-fhn> LinkHints.activateModeToOpenInNewTab match=".storylink"
``` 

Returns "No matching targets found" on all sites, except HN where it highlights all the links.

**Comment by gdh1995** [comment]
Confirmed. Very sorry!

v1.90.2 has a bug causing the "`host`" condition is "reverted", which means:
* `host="https://www.github.com"` means to exclude all GitHub pages.
* only `host="^(?!https://news\\.ycombinator\\.com/)"` can match `news.yc` and exclude other websites.

**Comment by gdh1995** [comment]
Hello, in recent commits there're too many changes so I need another tens of days to find new typos and logic issues, and then it seems a next version can't be released until this month ending.

Here're packages built from the latest code in my developing project: [vimium_c-1.90.2-firefox-c81a225-mod.zip](https://github.com/gdh1995/vimium-c/files/6767714/vimium_c-1.90.2-firefox-c81a225-mod.zip) and [vimium_c-1.90.2-chrome-c81a225-mod.zip](https://github.com/gdh1995/vimium-c/files/6767716/vimium_c-1.90.2-chrome-c81a225-mod.zip) . And you may test them by following steps in https://github.com/gdh1995/vimium-c/wiki/Test-a-zip-package .

**Comment by michielappelman** [comment]
Confirmed, fixed for Firefox with Vimium C v1.92.0. Thanks, @gdh1995!
